src/tests/dispatch-payload.test.ts

```diff
@@ -195,10 +212,11 @@ Deno.test("DispatchPayload with unhandled events", async (t) => {
       mock.assertSpyCalls(unhandledEventSpy, 0);
       mock.assertSpyCall(defaultSpy, 0, {
         args: [{
+          body: payload.body,
```

Comment:
This is related to a comment above where I mentioned that `body` is now passed into `function_executed` handlers.